trino-s3-proxy/pom.xml

```diff
@@ -127,6 +121,18 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>minio</artifactId>
```

Comment:
I see that Trino uses both the Adobe testcontainers ([here](https://github.com/trinodb/trino/blob/d8f7efa41e98966355ffdfb0adfbb923b5a70ec0/pom.xml#L316-L320)) as well as Minio under [trino-testing-containers](https://github.com/trinodb/trino/blob/d8f7efa41e98966355ffdfb0adfbb923b5a70ec0/testing/trino-testing-containers/src/main/java/io/trino/testing/containers/Minio.java#L44-L45)

They also seem to pin to a specific Minio release.

Is there any reason? Minio is AGPLv3 licensed whereas the Adobe one is Apache 2.0

I'm no expert in licensing so I'll leave it to you and @dain to comment. Just wondering whether there's a reason some of the Trino code doesn't use Minio, and whether those reasons may become an issue in this project too

Reply:
I'll let @dain respond to what Trino uses. Also @electrum might know.

Reply:
For us, the Adobe container is missing a lot of features. For example, it doesn't support any signing, etc. It accepts whatever requests you send.